GPCS4/SceModules/SceLibkernel/sce_kernel_pthread.cpp

```diff
@@ -177,8 +184,10 @@ void * PS4API scek_pthread_getspecific(pthread_key_t key)
 }
 
 
-int PS4API scek_pthread_key_create(void)
+int PS4API scek_pthread_key_create(pthread_key_t *key, void(*destructor)(void*))
```

Comment:
`destructor` is sysv_abi, to implement this function, we need a wrapper, like what we did for `scePthreadCreate`.